recipe_scrapers/forksoverknives.py

```diff
@@ -1,13 +1,47 @@
 # mypy: disallow_untyped_defs=False
+import re
+from typing import Dict, Optional, Tuple, Union
+
 from ._abstract import AbstractScraper
+from ._schemaorg import SchemaOrg
 from ._utils import normalize_string
 
 
+class fok_Schema(SchemaOrg):
+    def __init__(self, page_data, raw=False):
+        super().__init__(page_data)
```

Comment:
Ah, one more question: is there a reason that we have to override the `__init__` method here (and/or omit the `raw` argument when calling it?)

Reply:
Oops, that's a leftover from a dead-end experiment. Removed

Reply:
Ok, good catch - I've converted the pull request into a draft which disables merging from the pull request page until we convert it back into a 'ready' state.

I'd be fine with either: opening a new branch & pull request, or: you to amend those commits in this branch and force push (that's fine for development branches, generally).  Force-pushing over this branch is probably preferable if that's alright with you, because then we keep the discussion and review in one place.